packages/v-protocol/src/inbound-parser.test.ts

```diff
@@ -389,51 +389,34 @@ describe('PgPacketStream', function () {
     })
   })
 
-  describe('parses replication start message', function () {
```

Comment:
The "replication start" message is a postgres protocol message that Vertica does not have. This could have been removed a while ago, it's just dead code. The parser was still in place and this is a unit test so it continued to pass fine and not cause any issues